Snowflake.Data.Tests/UnitTests/SFResultSetArrowTest.cs

```diff
@@ -0,0 +1,86 @@
+﻿/*
+ * Copyright (c) 2012-2019 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using Apache.Arrow;
+using Apache.Arrow.Ipc;
+using NUnit.Framework;
+using Snowflake.Data.Core;
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    [TestFixture]
+    class SFResultSetArrowTest
+    {
+        [Test]
+        [Ignore("SFResultSetArrowTest")]
+        public void SFResultSetArrowTestDone()
+        {
+            // Do nothing;
+        }
+        
+        private RecordBatch PrepareRecordBatch(int rows)
+        {
+            var range = Enumerable.Range(1, rows);
+            return new RecordBatch.Builder()
```

Comment:
is it the same process of preparaing data as in previous test file? could we extract it to some helper file?

Reply:
I'm not sure how will it evolve after adding support of different column types. I cleaned it and moved to setup section.